src/main/java/fi/metatavu/metaform/server/rest/RealmsApiImpl.java

```diff
@@ -62,9 +63,9 @@
 
   @Inject
   private ReplyTranslator replyTranslator;
-  
+
   @Override
-  public Response createReply(String realmId, UUID metaformId, Reply payload, Boolean updateExisting) throws Exception {
+  public Response createReply(String realmId, UUID metaformId, Reply payload, Boolean updateExisting, String replyModeParam) throws Exception {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor this method to reduce its Cognitive Complexity from 19 to the 15 allowed. ([squid:S3776](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS3776))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=30&issues=AWPK8wyp4fWm9CZEX1S4&open=AWPK8wyp4fWm9CZEX1S4)

[ISSUE_KEY]: AWPK8wyp4fWm9CZEX1S4